wandb/sdk/wandb_init.py

```diff
@@ -379,6 +390,7 @@ def init(self):
                 self.teardown()
                 raise UsageError(error_message)
             run._set_run_obj(ret.run)
+            run._on_init()
```

Comment:
To keep the messages consistent, can you put run._on_init() before communicate_run

Reply:
sure thing